### PR TITLE
Add Ports explanation to troubleshootingp page

### PR DESCRIPTION
**Pull request by Kirbeerus**
## Describe your PR

_Summarize your changes here : Explanation of the ports a user may see open when they try to scan their domain.

## Checklist

- [ ] My PR is related to an opened issue : #
- [x] I've read the [contributing guidelines](/CleverCloud/documentation/blob/main/CONTRIBUTING.md)

## Reviewers
_Who should review these changes?_ @

### CODE REVIEW

File: content/doc/find-help/troubleshooting.md [hunk 1]
```diff
@@ -169,3 +169,16 @@ You can of course reach to <support@clever-cloud.com> if this page was not helpf
 
 Check the status of the services on [clevercloudstatus.com](https://www.clevercloudstatus.com/). The support team is also there to help you identify any potential errors or misconfiguration.
 {{% /details %}}
+
+{{% details title="A lot of port appear open after a scan of my domain" closed="true" %}}
```
Comment (cnivolle) [inline content/doc/find-help/troubleshooting.md:173]:
```suggestion
{{% details title="There are a lot of open ports, according to a port scan." closed="true" %}}
```

File: content/doc/find-help/troubleshooting.md [hunk 2]
```diff
@@ -169,3 +169,16 @@ You can of course reach to <support@clever-cloud.com> if this page was not helpf
...
+
+If you tried to scan your domain, you may not only have scanned your app, but also several public load balancers.
+Here is an explanation of the different port you may see.
+
+- Port 9999 : Zabbix use this port to monitor the hypervisor.
+- Ports 5000 to 6000 : The TCP redirection feature use these port to work. If you see the port in this range open but don't have the feature enabled, it means other app also present on the same hypervisor uses this feature. The load balancer link a specific port to an app, you don't have to worry about other people using this port to access your app.
+- Port 3000 : Juggernaut, the daemon torrent for the VM images use this port.
+- Ports 1080 and 1081 : They are necessary for the reverse-proxy Sozu to be able to redirect traffic to your application. Port 1080 is for HTTP and 1081 for HTTPS
```
Comment (cnivolle) [inline content/doc/find-help/troubleshooting.md:181]:
```suggestion
- Ports 1080 and 1081 : Those are necessary for the reverse-proxy Sōzu to be able to redirect traffic to your application. Port 1080 is for HTTP and 1081 for HTTPS
```

**Review by cnivolle — CHANGES_REQUESTED** [review]

File: content/doc/find-help/troubleshooting.md [hunk 3]
```diff
@@ -169,3 +169,16 @@ You can of course reach to <support@clever-cloud.com> if this page was not helpf
 
 Check the status of the services on [clevercloudstatus.com](https://www.clevercloudstatus.com/). The support team is also there to help you identify any potential errors or misconfiguration.
 {{% /details %}}
+
+{{% details title="There are a lot of open ports, according to a port scan." closed="true" %}}
+
+If you tried to scan your domain, you may not only have scanned your app, but also several public load balancers.
+Here is an explanation of the different port you may see.
+
+- Port 9999 : Zabbix use this port to monitor the hypervisor.
```
Comment (juliamrch) [inline content/doc/find-help/troubleshooting.md:178]: Fix grammar and add link to Zabbix homepage, so readers know what we are talking about.

```suggestion
- Port 9999 : [Zabbix](https://www.zabbix.com) uses this port to monitor the hypervisor.
```

File: content/doc/find-help/troubleshooting.md [hunk 4]
```diff
@@ -169,3 +169,16 @@ You can of course reach to <support@clever-cloud.com> if this page was not helpf
...
+- Ports 5000 to 6000 : The TCP redirection feature use these port to work. If you see the port in this range open but don't have the feature enabled, it means other app also present on the same hypervisor uses this feature. The load balancer link a specific port to an app, you don't have to worry about other people using this port to access your app.
```
Comment (juliamrch) [inline content/doc/find-help/troubleshooting.md:179]:
```suggestion
- Ports 5000 to 6000 : The TCP redirection feature uses these ports to work. If you notice a port within this range open but have not enabled the feature, it indicates that another app on the same hypervisor is using it. The load balancer assigns a unique port to each app, ensuring that your app is securely isolated. You can be assured that other users cannot access your app through this port.
```

**Review by juliamrch — COMMENTED** [review]
Thank you for this addition, very much needed!

I've added some suggestions to fix grammar and add clarity around the TCP feature. Feel free to commit them directly or to modify them if needed.

**Comment by Kirbeerus** [comment]
I have added the suggestions and corrected some mistakes, is it okay to merge?

File: content/doc/find-help/troubleshooting.md [hunk 5]
```diff
@@ -169,3 +169,16 @@ You can of course reach to <support@clever-cloud.com> if this page was not helpf
...
+{{% details title="There are a lot of open ports, according to a port scan." closed="true" %}}
+
+If you tried to scan your domain, you may not only have scanned your app, but also several public load balancers.
+Here is an explanation of the different port you may see.
+
+- Port 9999 : [Zabbix](https://www.zabbix.com) uses this port to monitor the hypervisor.
+- Ports 5000 to 6000 : The TCP redirection feature uses these ports to work. If you notice a port within this range open but haven't enabled the feature, it indicates that another app on the same hypervisor is using it. The load balancer assigns a unique port to each app, ensuring that your app is securely isolated. You can be assured that other users can't access your app through this port.
+- Port 3000 : Juggernaut, the daemon torrent for the VM images use this port.
+- Ports 1080 and 1081 : Those are necessary for the reverse-proxy Sōzu to be able to redirect traffic to your application. Port 1080 is for HTTP and 1081 for HTTPS
+- Port 80 and 443 : Your app use these ports to receive HTTP and HTTPS traffic
```
Comment (juliamrch) [inline content/doc/find-help/troubleshooting.md:182]:
```suggestion
- Port 80 and 443 : Your app uses these ports to receive HTTP and HTTPS traffic
```

**Review by juliamrch — APPROVED** [review]

**Pull request merged**